include/gpio_wrapper.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "nrf_gpio.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+namespace edge::aidan {
+
+inline bool read_gpio(uint32_t pin_number)
```

Comment:
pls hpp/cpp instead of inline inline inline

(not stylistic, i dont want nrf_gpio.h to be in every single translation unit)

Reply:
i think its fine

Reply:
if stylistic sure, but I think poor cop out